src/com/android/jcserver/JCServer.java

```diff
@@ -22,13 +22,17 @@ public static void main(String[] args) {
         String targetName;
         Simulator simulator;
         String[] targetNames;
+        String pathToCapFiles = null;
```

Comment:
Do we need to set this to default path ?

Reply:
Done